src/common/collection.c

```diff
@@ -300,6 +379,7 @@ void dt_collection_reset(const dt_collection_t *collection)
   params->comparator = dt_conf_get_int("plugins/collection/rating_comparator");
   params->filter_flags = dt_conf_get_int("plugins/collection/filter_flags");
   params->sort = dt_conf_get_int("plugins/collection/sort");
+  params->sortSecondOrder = dt_conf_get_int("plugins/collection/sortSecondOrder");
```

Comment:
style here, all variable are lower-case in dt, so I would use `params->sort_second_order`.